src/main/java/org/jabref/gui/StateManager.java

```diff
@@ -28,6 +32,8 @@
 
 import com.tobiasdiez.easybind.EasyBind;
 import com.tobiasdiez.easybind.EasyBinding;
+import org.apache.commons.logging.Log;
```

Comment:
Must be 
```
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
```

Reply:
Thanks! Thats what you get for trusting Itellij to autoguess imports...